kano_login/login.py

```diff
@@ -310,12 +310,12 @@ def __init__(self, win):
 
         self.email_entry = self.labelled_entries.get_entry(0)
         self.email_entry.set_text(user_email)
-        self.email_entry.connect("key-release-event", self.activate)
+        self.email_entry.connect('key-release-event', self.activate)
 
-        self.button = KanoButton(_("Reset password").upper())
+        self.button = KanoButton(_("RESET PASSWORD"))
```

Comment:
As a note: removing the `.upper()` from here assumes the translators will know to keep the styling..